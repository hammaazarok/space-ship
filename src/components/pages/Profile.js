import React from 'react';

const Profile = () => (
  <div>My Profile</div>
);

export default Profile;
