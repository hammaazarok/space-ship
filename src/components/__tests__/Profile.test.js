import React from 'react';
import Renderer from 'react-test-renderer';
import Profile from '../pages/Profile';

it('renders correctly', () => {
  const tree = Renderer
    .create(<Profile />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
