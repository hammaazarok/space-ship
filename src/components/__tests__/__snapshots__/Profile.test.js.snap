// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="profile"
>
  <div
    className="profile-missions"
  >
    <h2>
      My Missions
    </h2>
    <div
      className="list-group"
      onKeyDown={[Function]}
    />
  </div>
  <div
    className="profile-rockets"
  >
    <h2>
      My Rockets
    </h2>
    <div
      className="list-group"
      onKeyDown={[Function]}
    />
  </div>
</div>
`;
